test/restart/agent_config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "agent": {
+        "debug": true
+    }
```

Comment:
This agent config will fail to start. Is that intentional?

Reply:
yes, the common module expects this file so it's just there for that